sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,187 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import Any, Dict, Mapping, Optional, Iterable, List
+from datetime import datetime
+from .._generated.v7_0 import models
+from ._models import (
+    Certificate,
+    CertificateBase,
+    DeletedCertificate,
+    CertificatePolicy,
+    Issuer,
+    IssuerBase,
+    Contact,
+    CertificateOperation,
+)
+
+
+class CertificateClient:
+    @staticmethod
+    def create_config(**kwargs):
+        pass  # TODO
+
+    def __init__(self, vault_url, credentials, config=None, api_version=None, **kwargs):
+        pass
+
+    @property
+    def vault_url(self):
```

Comment:
Once you rebase onto master you can delete these methods and inherit `_KeyVaultClientBase` instead.